### ISSUE TITLE
What is YOLOv11's training configurations to reproduce the reported COCO results?

### ISSUE THREAD

**Issue by sunsmarterjie** [ultralytics/ultralytics#18903]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I have reviewed the related issues, and I believe my training hyperparameters are correct. However, I am unable to reproduce the YOLOv11 results on COCO. The best results I achieved are as follows:  YOLOv11n: 39.3; YOLOv11s: 46.6; YOLOv11m: 51.0; YOLOv11l: 53.1; YOLOv11x: 54.5.  

Below are my training hyperparameters and logs files. I would greatly appreciate it if you could help me identify the cause (hyperparameters/log). 

YOLOv11n: ([args.txt](https://github.com/user-attachments/files/18550416/args.txt)/[results.csv](https://github.com/user-attachments/files/18550417/results.csv)); YOLOv11s: ([args.txt](https://github.com/user-attachments/files/18550418/args.txt)/[results.csv](https://github.com/user-attachments/files/18550419/results.csv)); YOLOv11m: ([args.txt](https://github.com/user-attachments/files/18550420/args.txt)/[results.csv](https://github.com/user-attachments/files/18550422/results.csv)); YOLOv11l: ([args.txt](https://github.com/user-attachments/files/18550424/args.txt)/[results.csv](https://github.com/user-attachments/files/18550426/results.csv)); YOLOv11x: ([args.txt](https://github.com/user-attachments/files/18550432/args.txt)/[results.csv](https://github.com/user-attachments/files/18550439/results.csv)). 

If needed, I can also provided the trained models.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @sunsmarterjie, thank you for your interest in Ultralytics 🚀! We appreciate your detailed question and the attached files.

For training and reproducing YOLOv11 results on COCO, precise configurations and environmental factors are critical. While waiting for an Ultralytics engineer to review and assist, you may find the following resources helpful:

1. Check out our [Docs](https://docs.ultralytics.com/) for comprehensive guides, including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) examples.
2. For training tips and best practices, see our [Model Training Tips](https://docs.ultralytics.com/guides/model-training-tips/).

In the meantime, consider verifying or updating your environment. Ensure you are using the latest YOLO version and dependencies:

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Minimum Reproducible Example (MRE) 🐛

To help further, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) if you haven't already. This would include:
- Full training code or command used
- Detailed environment information (hardware, software, versions)
- Key hyperparameters (batch size, learning rate, epochs, etc.)

## Environment Options

YOLO can be run in any of the following up-to-date verified environments, which we also recommend for consistent replicability:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. Refer to the [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/).
- **Amazon** Deep Learning AMI. Refer to the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).
- **Docker Image**. Refer to the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>.

## Community Support 💬

You might also want to join the Ultralytics community for real-time interaction or in-depth discussions:
- [Discord](https://discord.com/invite/ultralytics) 🎧 for live chat
- [Discourse](https://community.ultralytics.com/) for detailed discussions
- [Subreddit](https://reddit.com/r/Ultralytics) for sharing knowledge and insights

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, our [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing, verifying the correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

This is an automated response 😊, and an Ultralytics engineer will review your question and assist soon. Thank you for your patience!

**Comment by Y-T-G** [comment]
Do you have albumentations installed?

**Comment by sunsmarterjie** [comment]
Thank you for your reply. May I confirm that this library will improve performance, correct? Are there any other libraries that need to be installed? If possible, could you share the runtime environment details? Thanks!

**Comment by Y-T-G** [comment]
Installing albumentations would enable some augmentations. AFAIK it was used for official training.

**Comment by sunsmarterjie** [comment]
Thanks, I will try that

**Comment by glenn-jocher** [comment]
You're welcome! Let us know if you encounter any issues or need further assistance.
